mappings/net/minecraft/entity/decoration/ItemFrameEntity.mapping

```diff
@@ -8,13 +8,13 @@ CLASS net/minecraft/class_1533 net/minecraft/entity/decoration/ItemFrameEntity
 		ARG 1 type
 		ARG 2 world
 		ARG 3 pos
-		ARG 4 direction
+		ARG 4 facing
 	METHOD <init> (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)V
 		ARG 1 world
 		ARG 2 pos
-		ARG 3 direction
+		ARG 3 facing
 	METHOD method_33339 isGlowItemFrame ()Z
-	METHOD method_33340 getStack ()Lnet/minecraft/class_1799;
+	METHOD method_33340 getAsItemStack ()Lnet/minecraft/class_1799;
```

Comment:
Gets this frame itself as an item stack?

Reply:
Yes